presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,6 +171,55 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules govern who is considered an owner of a schema. The user is granted
+the privileges based on the first matching rule read from top to bottom. If no
+rule matches, the user is not considered an owner of the schema. Each rule is
+composed of the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema names. Defaults to ``.*``.
+* ``owner`` (required): boolean indicating whether the user is to be considered
+  an owner of the schema. Defaults to ``false``.
+
+For example, to provide ownership of all schemas to user ``admin``, treat all
+users as owners of ``default`` schema and prevent user ``guest`` from ownership
+of any schema, you can use the following rules:
+
+.. literalinclude:: schema-access.json
```

Comment:
Personally I would just inline the code snippet in a code block .. wdyt @electrum ?